src/lib/ERC4337/utils/ERC4337Storage.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+library ERC4337Storage {
+    bytes32 internal constant SLOT = keccak256(abi.encode(uint256(keccak256("mage.ERC4337")) - 1));
+
+
+    struct Layout {
+        /// @dev This chain's EntryPoint contract address
+        address entryPoint;

```

Comment:
Good point. I also now realize it is less gas efficient and doesn't need to be in namespace since it won't be in storage when converted to immutable. Resolved  in f6eb172